_reports/adhikasp.md

```diff
@@ -40,48 +40,48 @@ blog: https://adhikasp.github.io
 activity:
  - 0:
    - repo: coala
-   - link: https://github.com/coala/coala/pull/4412
+   - link: https://github.com/coala/coala/pull/4412/commits
```

Comment:
detail should be `aspectModule: Add ``get()`` method` (commit head)
link should be https://github.com/coala/coala/pull/4412/commits/66d0276f0a45f221bc2d880a4fd5d4c63f6d119e